app/celery/research_mode_tasks.py

```diff
@@ -153,6 +217,17 @@ def _fake_sns_s3_callback(filename):
     return json.dumps({"Type": "Notification", "MessageId": "some-message-id", "Message": message_contents})
 
 
+@notify_celery.task(bind=True, name="create-fake-letter-callback", max_retries=3, default_retry_delay=60)
+def create_fake_letter_callback(self, notification_id: uuid.UUID, billable_units: int, postage: str):
+    try:
+        send_letter_response(notification_id, billable_units, postage)
+    except Exception:
```

Comment:
i dont know if this is a bit too wide-reaching an exception to catch - should this just be `HTTPError`? worried we might miss actual errors with json coding etc